applications/nrf_desktop/README.rst

```diff
@@ -176,8 +176,41 @@ When configuring HEAP, make sure that the values for the following options match
     This backend is used by default in Zephyr.
     For more information, refer to Zephyr's documentation about :ref:`zephyr:heap_v2`.
 
-HID mouse data forwarding
-=========================
+HID data handling
+=================
+
+HID data can be exchanged using one of the following HID report types:
+
+* HID input report,
+* HID output report,
+* HID feature report.
+
+nRF Desktops uses all of these report types.
+See sections below for details about handling given HID report type in nRF Desktop application.
+
+HID input reports
+-----------------
+
+The HID input reports are used to transmit information about user input from nRF Desktop device to a host.
+The user input can be, for example, button press or mouse motion.
+
+The nRF Desktop supports the following HID input reports:
+
+* HID mouse report,
+* HID keyboard report,
+* HID consumer control report,
+* HID system control report.
+
+Every of these reports uses predefined report format and is dedicated to provide given information.
+For example, the mouse motion is forwarded as HID mouse report.
+
+Given nRF Desktop device supports selected subset of the HID input reports.
```

Comment:
s/nRF Desktop/the nRF Desktop
probably in many places

Reply:
fixed